packages/salesforcedx-utils-vscode/src/telemetry/telemetry.ts

```diff
@@ -52,25 +57,33 @@ export class TelemetryBuilder {
   }
 }
 
+export class TelemetryProvider {
```

Comment:
This should be renamed TelemetryServiceProvider, since it is now offering instances of TelemetryService.